### PR TITLE
Increased visibility of variant callers's "Pass" or "Filtered" on SNVs and SVs variants pages (both RD and cancer variants)

### LINKED ISSUE #5011: Caller badges for cancer variantS views
Show multiple callers for cancer variantS views. They are now shown as hovers. Currently only Balsamic SV VCFs have multiple callers, but SNVs will soonish as well.

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR adds a functionality or fixes a bug.
- closes #5011

### Main branch

image

### This branch

image

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
### Check the following pages:
- SNVs
- Cancer SNVs
- SVs
- Cancer SVs

###
- Check the export of all variant types

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

File: scout/server/blueprints/variants/templates/variants/variants.html [hunk 1]
```diff
@@ -55,8 +55,9 @@
          <thead class="thead table-light">
             <tr>
               <th style="width:2%"></th>
-              <th style="width:12%" title="Rank position">Rank</th>
+              <th style="width:5%" title="Rank position">Rank</th>
               <th style="width:5%" title="Rank score">Score</th>
+              <th style="width:7%">Callers</th>
```
Comment (dnil) [inline scout/server/blueprints/variants/templates/variants/variants.html:60]: We chase these % so hard: I don't think this is useful enough here. More clinvar detail, quality measures, more in silico predictor results, more gene details all would go before in mind: you would not dismiss or be interested in a variant based on the RD SNV caller these days.

**Review by dnil — CHANGES_REQUESTED** [review]
I understand the use case for cancer and cancersv: given present issues it is well worth the real estate. For RD SVs, I don't feel I know the bottom line. It would be rather helpful sometimes, mostly occupy space/attention other times. Let's try it out!

For RD SNVs this would be distracting/taking too much space in relation to its use: 
Screenshot 2024-11-13 at 09 14 49

File: scout/server/blueprints/variants/templates/variants/variants.html (hunk 1 above)
Comment (northwestwitch) [reply scout/server/blueprints/variants/templates/variants/variants.html:60]: That's fine, we can remove that callers colum completely

**Comment by northwestwitch** [comment]
> I understand the use case for cancer and cancersv: given present issues it is well worth the real estate. For RD SVs, I don't feel I know the bottom line. It would be rather helpful sometimes, mostly occupy space/attention other times. Let's try it out!
> 
> For RD SNVs this would be distracting/taking too much space in relation to its use: Screenshot 2024-11-13 at 09 14 49

I've reverted the SNVs template now!

**Review by dnil — APPROVED** [review]
Nice! Let's go!

**Pull request merged**